lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py

```diff
@@ -1531,31 +1533,31 @@ def create_default_ipv6_flows(self, port_idx=0):
         """
         # UL Flow description #1
         ul_flow1 = {
-            "ipv6_dst": "5546:222:2259::226",  # IPv6 destination address
+            "ipv6_dst": "3001::2",  # IPv6 destination address
```

Comment:
These are complete IPv6 destination address or only the prefixes?

Reply:
Complete IPv6 address